src/middlewared/middlewared/plugins/catalogs_linux/items.py

```diff
@@ -151,7 +151,7 @@ def item_version_details(self, version_path):
         for key, filename, parser in (
             ('chart_metadata', 'Chart.yaml', yaml.safe_load),
             ('schema', 'questions.yaml', yaml.safe_load),
-            ('app_readme', 'app-readme.md', str.strip),
+            ('app_readme', 'app-readme.txt', str.strip),
```

Comment:
@Ornias1993 can we please use `markdown.markdown` instead ? We will be making relevant changes on the UI side to accommodate this - why it would be nicer is because it gives us more leverage wrt formatting which the chart devs can control and we can standardize like on chart release upgrade summary as well.

Reply:
@sonicaj Ofcoarse thats the logical alternative :)